docs/references/index.md

```diff
@@ -0,0 +1,23 @@
+---
+title: References
+---
+
+# References
+
+## Languages
+
+- [HTTP API](/http-api)
```

Comment:
I think we need a little blurb for each of these. just to give a little context for the link